### PR TITLE
#1643 - Individual resources can now be set to download or open in new window 

### PR DESCRIPTION
**Pull request by ryan-lynch**
#1643 - Added a forceDownload property to resourceItems which defaults to false, all items now open in a new window by default. This setting can be changed to download for individual resource items. Slight tweak to the code executed on click of a resource, it now looks for the forcedownload option, rather than forcing documents to be downloaded. Version bump

### CODE REVIEW

File: templates/resources.hbs [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	<div class="resources-item-container">
 	{{#each resources}}
 	<div class="resources-item drawer-item {{_type}}">
-		<button class="base resources-item-open drawer-item-open" data-type="{{_type}}" data-filename="{{filename}}" data-href="{{_link}}" tabindex="0" role="button" aria-label="{{lookup ../model._filterButtons _type}}. {{../model.itemAriaExternal}}. {{{title}}}. {{{description}}}">
+		<button class="base resources-item-open drawer-item-open" data-type="{{_type}}" data-filename="{{filename}}" data-href="{{_link}}" data-forceDownload="{{forceDownload}}" tabindex="0" role="button" aria-label="{{lookup ../model._filterButtons _type}}. {{../model.itemAriaExternal}}. {{{title}}}. {{{description}}}">
```
Comment (moloko) [inline templates/resources.hbs]: could you lowercase `data-forceDownload` to match the JS code? I know it works as is but I think it should be consistent

File: properties.schema [hunk 2]
```diff
@@ -148,6 +148,15 @@
                         "inputType": { "type": "Select", "options": ["document", "media", "link"]},
                         "validators": ["required"]
                       },
+                      "forceDownload": {
```
Comment (moloko) [inline properties.schema]: the standard we usually try to follow is that configuration properties start with an underscore, would you mind changing this to `_forceDownload` to keep with the convention?

File: templates/resources.hbs (hunk 1 above)
Comment (tomgreenfield) [inline templates/resources.hbs]: Suggest `data-force-download` to match data attributes convention.

Comment (ryan-lynch) [inline templates/resources.hbs]: Yes of course, will amend.

**Comment by ryan-lynch** [comment]
Thanks for pointing that out @tomgreenfield I have now addressed the whitespace issue.

**Comment by moloko** [comment]
@tomgreenfield I _think_ it will only affect someone who's currently running v2.0.6 of resources in a non-IE browser. so the scope of the impact is quite small.

You could equally argue that reversing the logic so that it would always download if `_forceDownload` wasn't set would be a breaking change from anyone coming from v2.0.5 (or earlier)!

File: example.json [hunk 3]
```diff
@@ -23,19 +23,22 @@
...
+            "_link":"https://www.youtube.com/channel/UCW8SlSFuCc--B66Gf9fAEcQ",
+            "_forceDownload": false
         },
         {
             "_type":"link",
             "title":"Community Site",
             "description":"Click here to view our community site",
-            "_link":"https://community.adaptlearning.org"
+            "_link":"https://community.adaptlearning.org",
+            "_forceDownload": true
```
Comment (moloko) [inline example.json]: could you set this to `false`?

**Pull request merged**